csi/volumeutils.py

```diff
@@ -1010,6 +1010,10 @@ def mount_glusterfs(volume, mountpoint, is_client=False):
             use_gluster_quota = True
         secret_private_key = "/etc/secret-volume/ssh-privatekey"
         secret_username = os.environ.get('SECRET_GLUSTERQUOTA_SSH_USERNAME', None)
+        
+        # SSH into only first host in volume['g_host'] entry
+        g_host = volume['g_host'].split(',')[0]
```

Comment:
Select a host that is reachable when multiple hosts provided.

For example,

```
def reachable_host(hosts):
    for host in hosts:
        host = host.strip()
        if is_host_reachable(host, 22):
            return host

    return None

g_host = reachable_host(volume['g_host'].strip().split(','))
if g_host is None:
    # Error, All hosts are not reachable
```

Reply:
is_host_reachable() is defined in operartor/main.py should I import it or re-define. Is it port 20 or 22?
```bash

Reply:
You can move that function to kadalulib.

Reply:
Done